.github/workflows/lint.yaml

```diff
@@ -0,0 +1,22 @@
+name: lint
+​
+on: [push, pull_request]
+​
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+​
+    steps:
+      - uses: actions/checkout@v2
+​
+      - name: Set up Python
+        uses: actions/setup-python@v1
```

Comment:
Out of interest: why v1 rather than v2?

Reply:
That's a question for @jonathansick

Reply:
ha, I didn't realize v2 was out. Feel free to update :)